test/e2e/framework/ginkgowrapper/wrapper.go

```diff
@@ -37,8 +37,18 @@ type FailurePanic struct {
 	FullStackTrace string // A full stack trace starting at the source of the failure
 }
 
+const GINKGO_PANIC = `
```

Comment:
Please don't export this constant. We don't want to expose it as API for the ginkgowrapper package.

Reply:
Looks good now.